cmd/minikube/cmd/start_flags.go

```diff
@@ -469,9 +470,15 @@ func getCNIConfig(cmd *cobra.Command) string {
 
 func getNetwork(driverName string) string {
 	n := viper.GetString(network)
-	if !driver.IsQEMU(driverName) {
-		return n
+	if driver.IsQEMU(driverName) {
+		return validateQemuNetwork(n)
+	} else if driver.IsVFKit(driverName) {
```

Comment:
nit: technically we dont need "else"

Reply:
We cannot assume that if it is not QEMU it is vfkit. This is fragile and will lead to bugs when we another driver will use --network and will get the wrong validation from the vfkit case.